src/azure-cli-core/azure/cli/core/aaz/_arg.py

```diff
@@ -599,6 +600,20 @@ def __init__(self, fmt=None, **kwargs):
         super().__init__(fmt=fmt, **kwargs)
 
 
+class AAZFileUploadArg(AAZStrArg, AAZFileUploadType):
+    """Argument that accepts a file path and returns file content, file hander and file size"""
+
+    @property
+    def _type_in_help(self):
+        return "File Content"
+
+    def _build_cmd_action(self):
+        class Action(AAZFileUploadTypeArgAction):
+            _schema = self  # bind action class with current schema
+
+        return Action
```

Comment:
You can directly use the AAZFileArg with a new binary fmt to verify the file existance